tests/vmi_multus_test.go

```diff
@@ -55,9 +55,6 @@ var _ = Describe("Multus", func() {
 	virtClient, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
 
-	var detachedVMI *v1.VirtualMachineInstance
```

Comment:
@SchSeba @booxter , I have to make the multus tests resilient against the MAC address error because of the reserved MAC range for libvirt. Could you have a look if the tests look good?